index.js

```diff
@@ -1,11 +1,17 @@
 'use strict';
 const path = require('path');
+const os = require('os');
 const pathIsInside = require('path-is-inside');
 
 module.exports = (childPath, parentPath) => {
 	childPath = path.resolve(childPath);
 	parentPath = path.resolve(parentPath);
 
+	if (os.platform() === 'win32') {
```

Comment:
```suggestion
	if (process.platform === 'win32') {
```

Reply:
I'll make the change.

On a side note/question, I thought it is best practice to use `os.platform()` instead of directly reading `process.platform`?

Reply:
That's news to me. Where did you get that from? I've always used `process.platform`, and from having read the source of thousands of modules, I've never seen `os.platform()` being used. It doesn't matter much. I just prefer `process.platform` for consistency between my modules.

Reply:
Not sure, probably just incorrectly remember wherever I read that a long time ago. Thanks for the response! Honestly I've learned a lot from you and your modules. Appreciate it!

I made the requested change.